src/pudl/transform/params/ferc1.py

```diff
@@ -4372,6 +4415,245 @@
+                    "average_number_of_customers_per_month_small_or_commercial_billed": "commercial_billed_avg_customers_per_month",
+                    "average_number_of_customers_per_month_large_or_industrial": "industrial_total_avg_customers_per_month",
+                    "average_number_of_customers_per_month_large_or_industrial_billed": "industrial_billed_avg_customers_per_month",
+                    "average_number_of_customers_per_month_commercial_and_industrial_sales": "commercial_and_industrial_total_avg_customers_per_month",
+                    "average_number_of_customers_per_month_commercial_and_industrial_sales_billed": "commercial_and_industrial_billed_avg_customers_per_month",
+                    "average_number_of_customers_per_month_public_street_and_highway_lighting": "public_lighting_total_avg_customers_per_month",
+                    "average_number_of_customers_per_month_public_street_and_highway_lighting_billed": "public_lighting_billed_avg_customers_per_month",
+                    "average_number_of_customers_per_month_other_sales_to_public_authorities": "public_authorities_total_avg_customers_per_month",
+                    "average_number_of_customers_per_month_other_sales_to_public_authorities_billed": "public_authorities_billed_avg_customers_per_month",
+                    "average_number_of_customers_per_month_sales_to_railroads_and_railways": "railroads_total_avg_customers_per_month",
+                    "average_number_of_customers_per_month_sales_to_railroads_and_railways_billed": "railroads_billed_avg_customers_per_month",
+                    "average_number_of_customers_per_month_interdepartmental_sales": "interdepartmental_total_avg_customers_per_month",
+                    "average_number_of_customers_per_month_interdepartmental_sales_billed": "interdepartmental_billed_avg_customers_per_month",
+                    "average_number_of_customers_per_month_provision_for_rate_refunds": "provision_for_rate_refund_total_avg_customers_per_month",
+                    "average_number_of_customers_per_month_provision_for_rate_refunds_billed": "provision_for_rate_refund_billed_avg_customers_per_month",
+                    "average_number_of_customers_per_month_sales_of_electricity_by_rate_schedules_unbilled": "total_unbilled_avg_customers_per_month",
+                    "average_number_of_customers_per_month_sales_of_electricity_by_rate_schedules": "total_billed_avg_customers_per_month",
+                    "average_number_of_customers_per_month_sales_of_electricity_by_rate_schedules_including_unbilled_revenue": "total_total_avg_customers_per_month",
+                    "average_kilowatt_hours_of_sales_per_customer_sales_of_electricity_by_rate_schedules": "total_billed_kwh_per_customer",
+                    "average_kilowatt_hours_of_sales_per_customer_sales_of_electricity_by_rate_schedules_including_unbilled_revenue": "total_total_kwh_per_customer",  # maybe we don't need this b/c calculated?
```

Comment:
so far we've defaulted to keeping the calculated values in there. we've discussed ripping them out during/after #2016